blocks/aside/aside.js

```diff
@@ -0,0 +1,124 @@
+  const twitterShare = `
+    <a href="${twitterUrl}"
+        onclick="return !window.open(this.href, 'Twitter', 'width=640,height=580')">
+        <span class="icon icon-social-twitter" />
+    </a>`;
+  createEl('div', { class: 'twitter-share' }, twitterShare, social);
+
+  // Facebook
+  const facebookUrl = `https://www.facebook.com/sharer/sharer.php?u=${pageUrl}&display=popup&ref=plugin&src=share_button`;
+  const facebookShare = `
+    <a href="${facebookUrl}"
+        onclick="return !window.open(this.href, 'Facebook', 'width=640,height=580')">
+        <span class="icon icon-social-facebook" />
+    </a>`;
+  createEl('div', { class: 'facebook-share' }, facebookShare, social);
+
+  // Email
+  const emailUrl = `mailto:?subject=${pageTitle}&body=Read this from Accenture Newsroom: ${pageUrl}`;
+  const emailShare = `
+    <a href="${emailUrl}"
+        target="_blank">
+        <span class="icon icon-social-email" />
+    </a>`;
+  createEl('div', { class: 'email-share' }, emailShare, social);
+
+  // Print
+  const printShare = `
+    <a href="javascript:void(0)"
+      onclick="window.print()">
+      <span class="icon icon-social-print" />
+    </a>`;
+  createEl('div', { class: 'print-share' }, printShare, social);
+
+  await decorateIcons(social);
+  const share = createEl('div', { class: 'share' }, social);
+  const shareTitle = createEl('h4', {}, 'Share');
+  share.prepend(shareTitle);
+  block.append(share);
+
+  // PDF Download button
+  const addPDF = getMetadata('pdf');
+  if (addPDF && (addPDF === 'true')) {
+    const pdfButton = createEl('a', { class: 'pdf-button button' }, 'DOWNLOAD PRESS RELEASE', share);
+    // Add the js2pdf script
+    // await loadScript('/scripts/jspdf.umd.min.js');
+    await loadScript('https://cdnjs.cloudflare.com/ajax/libs/jspdf/2.5.1/jspdf.umd.min.js');
+    await loadScript('/scripts/html2canvas.min.js');
+    pdfButton.addEventListener('click', async () => {
+      if (window.jspdf) {
+        await generatePDF(pageTitle);
+      }
+    });
+  }
+
+  // Tags
+  const industryTagValues = getMetadata('industry');
+  const subjectTagValues = getMetadata('subject');
+  const industryEl = createEl('div', { class: 'industry' }, '<h4>INDUSTRY TAGS</h4>');
+  const subjectEl = createEl('div', { class: 'subject' }, '<h4>SUBJECT TAGS</h4>');
+
+  const industryUl = createEl('ul', {}, '', industryEl);
+  industryTagValues.split(',').forEach((industryTag) => {
+    const cleanedUpValue = industryTag.trim().toLowerCase().replace(/[\W_]+/g, '-');
+    const link = createEl('a', { href: `/industry/${cleanedUpValue}` }, industryTag);
+    createEl('li', { class: 'industry-tag' }, link, industryUl);
+  });
+
+  const subjectUl = createEl('ul', {}, '', subjectEl);
+  subjectTagValues.split(',').forEach((subjectTag) => {
+    const cleanedUpValue = subjectTag.trim().toLowerCase().replace(/[\W_]+/g, '-');
+    const link = createEl('a', { href: `/industry/${cleanedUpValue}` }, subjectTag);
```

Comment:
typo.. please use the `/subjects/` for subject links

Reply:
done